src/routes/redirects.ts

```diff
@@ -112,10 +123,17 @@ const redirectToMobile =
 
     const url = new URL(req.raw.url!, 'http://localhost');
     await sendRedirectAnalytics(con, req, res, '/mobile');
-    if (isAndroid({ req })) {
+
+    const ua = uaParser(req.headers['user-agent']);
+    const os = ua.os.name?.toLowerCase();
+    if (os?.includes('android')) {
       return redirectToAndroid({ req, res });
     }
 
+    if (os?.includes('ios')) {
```

Comment:
does ios also includes ipad, not sure how the lib returns names?

Reply:
yes it should